# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libcap/hotfix-psx_calls.patch.alpha
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libcap-2.75/psx/psx_calls.c.vanilla	2025-03-23 15:13:27.626869769 +0000
+++ libcap-2.75/psx/psx_calls.c	2025-03-23 15:13:54.640732340 +0000
@@ -34,7 +34,7 @@
  * support here.
  */
 #if defined(__x86_64__) || defined(__i386__)	   \
-    || defined(__arm__) || defined(__aarch64__)	   \
+    || defined(__alpha__) || defined(__arm__) || defined(__aarch64__)	   \
     || defined(__mips__) || defined(__loongarch__) \
     || defined(__powerpc__) || defined(__s390__) || defined(__riscv)
 
